### ISSUE TITLE
[Question] Unable to edit channel settings in Bot Portal

### ISSUE THREAD

**Issue by asubbotin** [Microsoft/BotBuilder#2949]
Hello,

I can't edit my channel settings in bot. The buttons "Save", "Delete Channel" is disabled http://clip2net.com/s/3LkNRQB
![94cfc-clip-75kb](https://user-images.githubusercontent.com/3075047/27187405-fde51366-519f-11e7-9b3e-7ca0a951b8aa.png)

Can you explain why?

Thanks.

**Comment by nwhitmont** [comment]
@asubbotin The "Save" button only becomes active after all the required form fields are populated. Its a form of input validation.

**Comment by asubbotin** [comment]
@nwhitmont All form fields are populated, they are simply сolored for safety reasons.

**Comment by nwhitmont** [comment]
@asubbotin I'm unable to reproduce the error.  When I entered data in all the form fields for the FB channel config, the "Save" button was active (blue) and I was able to click & submit the form. (Form submission failed of course since this is fake input data. 
![fb channel config](https://user-images.githubusercontent.com/3075047/27192167-2d86529c-51af-11e7-859b-4d330298933c.png)


Maybe try logging out of bot portal and log back in and try again? Let me know if that works.
